### ISSUE TITLE
[Select] selected option is unreadable for dark-themed UIs in Firefox

### ISSUE THREAD

**Issue by jendowns** [carbon-design-system/carbon#4241]
## Environment

Mac OS 10.14.6
Firefox 69.0.2

## Detailed description

*NOTE: this is in Firefox*

After selecting an option with the `Select` component, the option is unreadable in Firefox:

<img width="728" alt="Screen Shot 2019-10-07 at 11 08 34 AM" src="https://user-images.githubusercontent.com/9057921/66328740-d2549900-e8f2-11e9-9bd4-e2c9fef0ffc1.png">

## Steps to reproduce the issue

1. Open Firefox
2. Go to the storybook (http://react.carbondesignsystem.com/?path=/story/select--default)
3. Select the `g100` Carbon Theme in the bottom panel
4. Open + select an option in the `Select` component. While the `Select` is still active, you will see that the text is black and unreadable.
